Source/WebCore/platform/sql/SQLiteStatementAutoResetScope.h

```diff
@@ -45,8 +45,8 @@ class SQLiteStatementAutoResetScope {
     bool operator!() const { return !m_statement; }
 
     SQLiteStatement* get() { return m_statement.get(); }
-    SQLiteStatement* operator->() { return m_statement.get(); }
     SQLiteStatement& operator*() const { ASSERT(m_statement); return *m_statement; }
+    CheckedPtr<SQLiteStatement> operator->() { return m_statement; }
```

Comment:
We typically don't change the return type of a function to be a smart pointer unless it's a protected* variant.